tests/dummy_book/mod.rs

```diff
@@ -38,12 +38,16 @@ impl DummyBook {
         self
     }
 
-    /// Write a book to a temporary directory using the provided settings.
-    pub fn build(&self) -> Result<TempDir> {
-        let temp = TempFileBuilder::new()
+    pub fn empty(&self) -> Result<TempDir> {
```

Comment:
This seems a little out of place, since it is no longer actually building the dummy book. That is, this method doesn't seem associated with the class DummyBook. Is there a reason that a test that needs an empty directory can't just call `TempFileBuilder` directly? That is done in quite a few other places. If we really need to shorten that, we can perhaps find a way to share it.

Reply:
Now creating the temporary directory directly.